content/guides/hcd/design-concepts/simple-is-hard.md

```diff
@@ -12,7 +12,7 @@ primary_image: hcd-design-concepts
 Leadership and clients should not expect a design solution to be completely finished in a month, especially if the team is working on multiple projects either together or apart, but rationally defining the term of the design phase can help.
 
 
-### Making decisions
+## Making decisions
```

Comment:
The code fix still works with the `h3` unless there is another reason we changed this an `h2`?

Reply:
This was from a test, should have left it as is.